.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @dslarm @isuruf @peterjc
\ No newline at end of file
+* @MementoRC @dslarm @isuruf @peterjc
```

Comment:
I think I'll take myself off this list (can do it now if you like, but afterwards is fine).

Reply:
Alrighty